packages/common/infra/src/livedata/index.ts

```diff
@@ -132,6 +133,43 @@ export class LiveData<T = unknown> implements InteropObservable<T> {
     return data;
   }
 
+  /**
+   * @example
+   * ```ts
+   * LiveData.fromYjs(ymap, ymap => ymap.toJSON(), {
+   *   deep: true,
+   * });
+   * ```
+   */
+  static fromYjs<T extends YAbstractType<any>, N>(
```

Comment:
I don't think the LiveData should be aware of what is yjs and yjs type.

`fromYjs` better to be made as an helper function somewhere else